common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -623,69 +643,74 @@ private void processSubscribe(ChannelHandlerContext ctx, MqttSubscribeMessage mq
             String topic = subscription.topicName();
             MqttQoS reqQoS = subscription.qualityOfService();
             try {
-                switch (topic) {
-                    case MqttTopics.DEVICE_ATTRIBUTES_TOPIC: {
-                        processAttributesSubscribe(grantedQoSList, topic, reqQoS, TopicType.V1);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_ATTRIBUTES_SHORT_TOPIC: {
-                        processAttributesSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_ATTRIBUTES_SHORT_JSON_TOPIC: {
-                        processAttributesSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2_JSON);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_ATTRIBUTES_SHORT_PROTO_TOPIC: {
-                        processAttributesSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2_PROTO);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_RPC_REQUESTS_SUB_TOPIC: {
-                        processRpcSubscribe(grantedQoSList, topic, reqQoS, TopicType.V1);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_RPC_REQUESTS_SUB_SHORT_TOPIC: {
-                        processRpcSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_RPC_REQUESTS_SUB_SHORT_JSON_TOPIC: {
-                        processRpcSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2_JSON);
-                        activityReported = true;
-                        break;
-                    }
-                    case MqttTopics.DEVICE_RPC_REQUESTS_SUB_SHORT_PROTO_TOPIC: {
-                        processRpcSubscribe(grantedQoSList, topic, reqQoS, TopicType.V2_PROTO);
-                        activityReported = true;
-                        break;
+                if (sparkPlugSessionHandler != null) {
```

Comment:
unreachable code. You already have check `if (sparkPlugSessionHandler != null)` in the `private void processPublish(ChannelHandlerContext ctx, MqttPublishMessage mqttMsg)`